cmdstanpy/model.py

```diff
@@ -459,19 +461,29 @@ def sample(
             try:
                 import tqdm
 
-                # These need update if CmdStan changes the defaults
-                # refresh every 0.5% --> 200 updates
+                # need 200 progress bar updates - 100 per warmup, sampling
+                # refresh = total iters * (1/200)
+                default_warmup = 1000
+                default_sampling = 1000
+                default_total = default_warmup + default_sampling
                 if warmup_iters is None and sampling_iters is None:
-                    refresh = 10  # 2000 // 200 => 0.5 %
+                    refresh = int(math.floor(default_total * 0.005))
```

Comment:
I think `int(default_total * 0.005)` is enough, it will automatically floor the value.